packages/compass-crud/src/components/insert-document-dialog.tsx

```diff
@@ -36,20 +37,48 @@ const documentViewContainer = css({
   marginTop: spacing[3],
 });
 
+export type InsertDocumentDialogProps = InsertCSFLEWarningBannerProps & {
+  closeInsertDocumentDialog: () => void;
+  toggleInsertDocumentView: (view: 'JSON' | 'List') => void;
+  toggleInsertDocument: (view: 'JSON' | 'List') => void;
```

Comment:
Should here also be a table? Like `view: 'List' | 'JSON' | 'Table'`

Reply:
We don’t have a table view in the insert modal, so `'JSON' | 'List'` is the right type